containers/0_Introductory/README.md

```diff
@@ -1,7 +1,50 @@
-# Introductory Container Examples
+# Introductory Python Container
+If you'd like to utilize a newer version of Python than what is installed in CCR's software repository or your workflow requires the use of GPUs, utilizing Python in a container environment is the ideal choice.
 
- TODO: write me
+This tutorial demonstrates how to download a Python Docker container, convert it to an Apptainer file, and run a simple script to extract the Python version.
 
-## How to use
+Note: Apptainer is only available on compute and compile nodes. Please refer to our [container documentation](https://docs.ccr.buffalo.edu/en/latest/howto/containerization/) for more information on using containers.
 
-TODO: write me
\ No newline at end of file
+## Using a specified version of Python with Apptainer
+1. Start an interactive job
+
+Apptainer is not available on the CCR login nodes and the compile nodes may not provide enough resources for you to build a container. We recommend requesting an interactive job on a compute node to conduct this build process. This will allow you to test your build after completion as well. Please refer to our documentation on [running jobs](https://docs.ccr.buffalo.edu/en/latest/hpc/jobs/#interactive-job-submission) for more information. This is provide as an example only and not all users will have access to the resources in this example:
+```
+$ salloc --cluster=ub-hpc --partition=general-compute --qos=general-compute --mem=32GB --time=02:00:00
```

Comment:
```suggestion
$ salloc --cluster=ub-hpc --partition=general-compute --qos=general-compute --mem=32GB --time=02:00:00 --no-shell
srun --jobid=JOBID_HERE --export=HOME,TERM,SHELL --pty /bin/bash --login
```
We have this incorrectly displayed throughout our examples, so let's start by correcting this one.  By using `--no-shell` we're ensuring the environment from the login node doesn't get copied over to the compute node.  Documentation on this can be found [here](https://docs.ccr.buffalo.edu/en/latest/hpc/jobs/#interactive-job-submission)